_posts/newsletters/2024-02-29-newsletter.md

```diff
@@ -0,0 +1,519 @@
+
+<a name="reads"></a>
+# **Interesting Reads, Videos, or Podcasts**
+
+* [The State of Diversity and Inclusion in the ASF Community: A Pulse Check](https://news.apache.org/foundation/entry/the-state-of-diversity-and-inclusion-in-the-asf-community-a-pulse-check) by The Apache Software Foundation Blog
+* [The White House Press Release: Future Software Should Be Memory Safe](https://www.whitehouse.gov/oncd/briefing-room/2024/02/26/press-release-technical-report/) by the White House Office of the National Cyber Director (ONCD)
+* [OpenFold Biotech AI Research Consortium releases SoloSeq and Multimer, an integrated protein Large Language Model with 3D structure generation](https://www.businesswire.com/news/home/20240219658831/en/OpenFold-Biotech-AI-Research-Consortium-releases-SoloSeq-and-Multimer-an-integrated-protein-Large-Language-Model-with-3D-structure-generation) by OpenFold Consortium
+* [US-RSE'23 Conference Proceedings Now Available](https://zenodo.org/communities/usrse23?q=&l=list&p=1&s=10) 
+* [Code for Thought: Make Science Reproducible (Again) - Rima-Maria Rahal, Peter Steinbach](https://www.youtube.com/watch?v=3z9Y3Id3YPg) by Peter Schmidt 
+* [Code for Thought: ByteSized RSE: Easy Does It - with easybuild - Jörg Saßmannshausen - ](https://codeforthought.buzzsprout.com/1326658/14361557-en-bytesized-rse-easy-does-it-with-easybuild-jorg-sassmannshausen) by Peter Schmidt
+* [Introducing JOSSCast: Open Source for Researchers](https://blog.joss.theoj.org/2024/01/introducing-josscast) by JOSSCast
+* [Code for Thought: Learning How to Learn (and Teach) - Barbara Oakley](https://codeforthought.buzzsprout.com/1326658/14419921-en-learning-how-to-learn-and-teach-barbara-oakley) by Peter Schmidt
+* [Unfair Discrimination Against Diamond Open Access Stifles Progress](https://council.science/current/blog/open-science-round-up-january-2024/) by Moumita Koley 
+
+
+
+
+
+<a name="outreach"></a>
+# **US-RSE Outreach**
+
+The Outreach Working Group is actively recruiting new co-chairs to take over its leadership! Volunteers are also needed to help with organizing workshops, birds of a feather (BoF), symposiums, etc. Roles span from organizer to reviewer to speaker and more. Here are some key projects that recent co-chairs, Miranda Mundt and Josh Teves, have led to benefit the overall US-RSE community:
+* Strategically planning and delivering the first-ever US-RSE conference in 2023
+* Organizing the submission of collaborative proposals to relevant conferences
+* Providing awesome SWAG to members
+* Assisting the overall US-RSE community with delineating between membership, communications, and outreach efforts
+
+If you are interested in contributing to a US-RSE hosted event, please reach out to the working group via the [#wg-outreach](https://app.slack.com/client/T8ZT4PJSW/CNR6E0N75) slack channel or [wg-outreach@us-rse.org](mailto:wg-outreach@us-rse.org).
```

Comment:
_sigh_ I would get rid of this. We've tried and can't find anyone.

Reply:
Thank you. This has been removed.